packages/data-point/lib/reducer-types/reducer-helpers/utils/index.js

```diff
@@ -0,0 +1,44 @@
+const last = require('lodash/last')
+
+/**
+ * used by ReducerDefault to check
+ * when the default should be used
+ * @param {*} value
+ * @return {boolean}
+ */
+function reducerOutputIsFalsy (value) {
```

Comment:
can you change to isFalsy.

The name could be aliased when required, but here it should just be called isFalsy. Also update the jsDoc

Reply:
Ok, I just wanted to be more specific because this doesn't return true for _all_ "falsy" values (including **false**)